packages/form/src/cli.ts

```diff
@@ -58,14 +125,20 @@ async function create() {
 		});
 	}
 
-	const relativeOutputPath = join(relativeOutputDir, "Form.tsx");
+	const relativeOutputPath = join(
+		relativeOutputDir,
+		toUseFramework.templateDir,
+	);
 	const outputPath = resolve(process.cwd(), relativeOutputPath);
```

Comment:
```suggestion
	const outputPath = resolve(process.cwd(), relativeOutputDir);
```

Ditto for this snippet. We should output to the user's chosen directory without the nested `react`, `preact`, etc